ios/MullvadVPN/AppDelegate.swift

```diff
@@ -437,6 +440,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         )
     }
 
+    private func getWipeSettingsOperation() -> AsyncBlockOperation {
```

Comment:
I think we should comment this function to explain in a few lines what are the expected conditions for a wipe, it took some time to test this properly, let's make sure our future selves profit from that time spent.

LGTM otherwise.